recipe/meta.yaml

```diff
@@ -31,18 +37,34 @@ requirements:
     - typing-extensions
 
 test:
+  source_files:
+    - tests
   imports:
     - google
   commands:
     - pip check
+    - pytest -v tests
   requires:
     - pip
+    - pytest
+    - absl-py
+    - grpcio
```

Comment:
```suggestion
```
This won't be necessary once https://github.com/AnacondaRecipes/google-ai-generativelanguage-feedstock/pull/1 is fixed.